.github/workflows/deploy-changed-users.yaml

```diff
@@ -43,9 +43,9 @@ jobs:
           if [ ! -z "$ERROR" ]
           then
             echo
-            echo Make sure you launch this job using the biomage devops tools, and not from GitHub.
-            echo Launching from GitHub directly is not supported to ensure cluster credentials are not
-            echo kept in the repository.
+            echo This workflow requires some secrets to run. The secrets can be set/rotated
+            echo by running `rotate-ci` from `biomage-utils`. The command has to be run by a
+            echo user with admin privileges for the repository.
```

Comment:
@aerlaut this is actually not true. This job is run automatically on the following conditions. Remove that part of the comment.
```
on:
  push:
    branches:
      - master
    paths:
      - 'infra/cluster_admins_*'
      - 'infra/cluster_users'
      - 'infra/biomage-read-only-group/**'
      - 'infra/cf-state-machine-role.yaml'
      
```

Reply:
This is not a comment. This is actually an error message that will appear if a secret is not set.

Reply:
Resolved with better text